firewallns.py

```diff
@@ -434,135 +434,195 @@ def build(
 
     # Block 05: Create temp nftables.conf file on Enabled PodNet
     # call rcc comms_ssh on enabled PodNet
-    try:
-        exit_code, stdout, stderr = comms_ssh(
-            host_ip=enabled,
-            payload=payload_create_nftables_file,
-            username='robot',
-        )
-    except CouldNotConnectException:
-        return False, messages[3050]
-
-    if exit_code != SUCCESS_CODE:
-        return False, f'{messages[3051]}\nExit Code: {exit_code}\nSTDOUT: {stdout}\nSTDERR: {stderr}'
+    response = comms_ssh(
+        host_ip=enabled,
+        payload=payload_create_nftables_file,
+        username='robot',
+    )
+    if response['channel_code'] != 200:
+        msg = f'{messages[3050]}\n ' \
+              f'channel_code: {response["channel_code"]}s.\n' \
+              f'channel_message: {response["channel_message"]}\n' \
+              f'channel_error: {response["channel_error"]}'
+        return False, msg
+    if response['payload_code'] != 0:
```

Comment:
```suggestion
    if response['payload_code'] != SUCCESS_CODE:
```

As `SUCCESS_CODE` is a global variable, we can continue to use it for the `payload_code`